lib/core/OAuth.ts

```diff
@@ -165,7 +167,7 @@ class OAuth {
     this.#credentials = {
       access_token: response.data.access_token,
       refresh_token: response.data.refresh_token || this.credentials.refresh_token,
-      expires: expiration_date
+      expires_in: expiration_date
```

Comment:
This is intentionally named `expires`. The expiration time is converted from seconds to a date, so it wouldn't make sense to have the original name there.

Reply:
Ohh okay, I was going to leave a comment but it seemed like a simple mistake.